pkgs/top-level/all-packages.nix

```diff
@@ -17839,6 +17839,8 @@ in
 
   airwave = callPackage ../applications/audio/airwave { };
 
+  akira-unstable = callPackage ../applications/graphics/akira/unstable.nix { };
```

Comment:
you'll still want `akira/default.nix`. The unstable convention is used when designating that this was cut from master, not at an official release